cmd/image-inspector.go

```diff
@@ -19,6 +19,7 @@ func main() {
 	flag.StringVar(&inspectorOptions.DockerCfg, "dockercfg", inspectorOptions.DockerCfg, "Location of the docker configuration file")
 	flag.StringVar(&inspectorOptions.Username, "username", inspectorOptions.Username, "username for authenticating with the docker registry")
 	flag.StringVar(&inspectorOptions.PasswordFile, "password-file", inspectorOptions.PasswordFile, "Location of a file that contains the password for authentication with the docker registry")
+	flag.StringVar(&inspectorOptions.OscapResultsArf, "openscap-results-arf", inspectorOptions.OscapResultsArf, "Run an OpenSCAP Scan on the inspected image and publish the reports. This value is also the reports location.")
```

Comment:
@pweil- @enoodle I don't have a preference in terms of names of the parameters and additional ones. For now what I am interested in is a way to specify the destination for openscap arf results. With a directory seems more flexible (which means that you have to come up with a standard name for the report in order to create it, read it to serve it through http). @pweil- can you add here a more consistent suggestion?

Maybe something like this?

```
image-inspector ... --scan-type=openscap --scan-reports-dir=/tmp/image-scan-reports
```